appgate/identity_provider.go

```diff
@@ -350,15 +338,7 @@ func readProviderFromConfig(d *schema.ResourceData, provider openapi.IdentityPro
 	if _, o := base.GetNotesOk(); o {
 		provider.SetNotes(base.GetNotes())
 	}
-	if v, ok := d.GetOk("display_name"); ok {
-		provider.SetDisplayName(v.(string))
-	}
-	if v, ok := d.GetOk("default"); ok {
-		provider.SetDefault(v.(bool))
-	}
-	if v, ok := d.GetOk("client_provider"); ok {
-		provider.SetClientProvider(v.(bool))
-	}
+
```

Comment:
Were those fields deprecated?

Reply:
yes from 5.1, I need to revisit the acceptance test for all identity providers to make them pass in 5.1, i will do it in another PR.